### PR TITLE
merge support for Zen2+, Thunder X, Bulldozer and GerminiLake

### PR DESCRIPTION
**Pull request by sidietz**
### PURPOSE
see title

### EFFECT OF CHANGE
none, just improved docs.

### TECHNICAL STEPS
none.

### NOTES FOR REVIEWERS
Approve or fix build pipeline, please.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,17 +19,21 @@ Getting and Building the Code
...
+    - git
 
 #### 2. Download the hdRpr source code
 
 You can use ```git``` to clone the repository.
+For modern, SMT or multi-threaded systems use:
 
 ```
-> git clone --recurse-submodules -j2 https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD
+> git clone --recurse-submodules -j64 https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD
```
Comment (bsavery) [inline README.md:32]: I would change this to say "git clone --recurse-submodules -j<num_cores> https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD"  where <num_cores> is the number of CPU cores, etc.

Comment (bsavery) [reply README.md:32]: Reason being that 64 threads even though there are only 5 submodules would oversubscribe an 8 core system.

Comment (d-croft) [reply README.md:32]: Not really. Modern schedulers and well written software, like servo or Firefox nightly uses more threads than physical cores, too.

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Getting and Building the Code
 You can use ```git``` to clone the repository.
 
 ```
-> git clone --recurse-submodules -j2 https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD
+> git clone --recurse-submodules -j64 https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD
```
Comment (d-croft) [inline README.md:32]: But I do agree, -j 16 should be sufficient.

**Review by d-croft — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (d-croft) [reply README.md:32]: Could someone post the jenkins log then I would investigate why it's failing.

Comment (hshakula) [reply README.md:32]: Your changes do not affect compilation or tests

**Pull request merged**